cms/templates/js/certificate-editor.underscore

```diff
@@ -0,0 +1,45 @@
+<form class="collection-edit-form certificate-edit-form">
+    <% if (error && error.message) { %>
+        <div class="certificate-edit-error message message-status message-status error is-shown" name="certificate-edit-error">
+          <%= gettext(error.message) %>
+        </div>
+    <% } %>
+    <div class="wrapper-form">
+        <fieldset class="collection-fields certificate-fields">
+            <legend class="sr"><%= gettext("Certificate Information") %></legend>
+            <div class="input-wrap field text required add-collection-name add-certificate-name <% if(error && error.attributes && error.attributes.name) { print('error'); } %>">
+                <label for="certificate-name-<%= uniqueId %>"><%= gettext("Certificate Name") %></label>
+                <% if (!_.isUndefined(id)) { %>
+                    <span class="certificate-id">
+                        <span class="certificate-label"><%= gettext('Certificate ID') %></span>
+                        <span class="certificate-value"><%= id %></span>
+                    </span>
+                <% } %>
+                <input id="certificate-name-<%= uniqueId %>" class="collection-name-input input-text" name="certificate-name" type="text" placeholder="<%= gettext("This is the Name of the Certificate") %>" value="<%= name %>">
+                <span class="tip tip-stacked"><%= gettext("Name or short description of the configuration") %></span>
```

Comment:
can you add an ID here i.e. ``` id="<%= uniqueId %>-tip"``` and then add an ```aria-describedby="<%= uniqueId %>-tip"``` to the ```<input>``` above.  This programmatically links the help text to the control, so it will be read when the control has focus.